recipes-kernel/nvidia-display-driver/nvidia-display-driver_34.1.1.bb

```diff
@@ -16,7 +16,7 @@ S = "${WORKDIR}/NVIDIA-kernel-module-source-TempVersion"
 B = "${S}"
 
 MODULES_MODULE_SYMVERS_LOCATION = "kernel-open"
-KERNEL_SPLIT_MODULES = "0"
+INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
```

Comment:
It wouldn't hurt to add a comment here to explain why this is being set.

Reply:
Agree, will add a comment there.

Reply:
@madisongh 
Just to double check with you. Should I add `PACKAGE_ARCH` in this recipe ?

Reply:
Oh, right, I missed that.  It's a kernel driver, so you should set `PACKAGE_ARCH = "${MACHINE_ARCH}"` in this recipe.

Reply:
I added it